stdlib/out_channel.mli

```diff
@@ -0,0 +1,133 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*             Xavier Leroy, projet Cristal, INRIA Rocquencourt           *)
+(*                                                                        *)
+(*   Copyright 2019 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+
+(** Output channels.
+
+    @since 4.10 *)
+
+type t = out_channel
+
+val stdout : t
+(** Standard output. *)
+
+val stderr : t
+(** Standard error. *)
+
+(** {1 General output functions} *)
+
+type open_flag =
+  | Append
+  | Create of int
+  | Exclusive
+  | Truncate
+  | Binary
+  | Non_blocking
+
+val open_ : ?flags:open_flag list -> string -> t
+(** [open_ ?flags fn] opens the file [fn] for writing, and returns a new output
+    channel on that file, positioned at the beginning of the file. The file is
+    truncated to zero length if it already exists. It is created if it does not
+    already exists. *)
+
+val with_file : ?flags:open_flag list -> string -> (t -> 'a) -> 'a
+(** [with_file ?flags filename f] opens the file named [filename] for writing
+    according to [flags], invokes [f] to process the contents of that file then,
+    once [f] has returned or triggered an exception, closes the file before
+    proceeding. *)
```

Comment:
I suggest to add `All errors arising during closing are ignored.` (idem for in in_channel.mli).

Reply:
Fixed.